README.md

```diff
@@ -91,6 +91,13 @@ In the root, create a file `.env` with the contents:
 SERVER_URL="<url to backend server>"
 ```
 
+If running the Flask backend in parallel with the frontend, the backend URL is
+`http://<IP address>:5000` 
+
+The IP address can be accessed by the `ifconfig` command for Unix or with `ipconfig`
+for Windows. The IP address should look similar to `192.168.x.y` where `x` and `y` 
+are numbers.
+
```

Comment:
Same here, not all networks use `192.168.*.*` -- some use `10.0.*.*` for example. Saying `x` and `y` are numbers is also redundant, since you can assume most people would know that IPv4 addresses consist of four numbers.

Reply:
I wanted to use 192.168.x.y as an example IP address if someone might be unfamiliar or not sure how it should look like

Reply:
Resolved in the following commit